src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -0,0 +1,431 @@
+# Electroencephalography (EEG)
+
+Support for EEG was developed as a [BIDS Extension Proposal](../06-extensions.md#bids-extension-proposals).
+Please cite the following paper when referring to this part of the standard in
+context of the academic literature:
+
+> Pernet, C. R., Appelhoff, S., Flandin, G., Phillips, C., Delorme, A., &
+> Oostenveld, R. (2018, December 6). BIDS-EEG: an extension to the Brain
+> Imaging Data Structure (BIDS) Specification for electroencephalography.
+> [https://doi.org/10.31234/osf.io/63a4y](https://doi.org/10.31234/osf.io/63a4y)
+
+The following example EEG datasets have been formatted using this specification
+and can be used for practical guidance when curating a new dataset.
+
+-   Single session per subject: [`eeg_matchingpennies`](https://doi.org/10.17605/OSF.IO/CJ2DR)
+-   Multiple sessions per subject: [`eeg_rishikesh`](https://doi.org/10.5281/zenodo.1490922)
+-   Combined with fMRI: [`eeg_rest_fmri`](https://osf.io/94c5t/files/)
+
+Further datasets are available from the [BIDS examples repository](https://github.com/bids-standard/bids-examples).
+
+## EEG recording data
+
+Template:
+
+```Text
+sub-<label>/
+    [ses-<label>]/
+      eeg/
+        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_eeg.<manufacturer_specific_extension>
+        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_eeg.json
+```
+
+The EEG community uses a variety of formats for storing raw data, and there is
```

Comment:
The EEG community uses a variety of formats for storing raw data, and there is no single standard that all researchers agree on. Original files (if different from the
supported formats) can be stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data) but for BIDS, EEG data MUST be stored in one of the following formats: